docs/cloud-edge/modules/_http_header_templates.mdx

```diff
@@ -97,26 +81,35 @@ This data is provided by [MaxMind GeoIP](https://www.maxmind.com/en/home).
 | `${.http.request.version}`      | The HTTP version used in the request.                                       |
 | `${.http.response.status_code}` | The status code of the response returned by the ngrok edge.                 |
 
-### IP Policy Template Variables {#ip-policy}
+#### IP Restrictions Variables {#ip-restrictions}
 
-Note: These values are only available when using the IP Policy module.
+These variables are only populated when using the IP Restrictions module.
 
 |                               |                                                                                               |
 | ----------------------------- | --------------------------------------------------------------------------------------------- |
-| `${.ip_policy.decision}`      | 'allow' if IP Policy module permitted the request to the upstream service, 'block' otherwise. |
+| `${.ip_policy.decision}`      | `allow` if IP Policy module permitted the request to the upstream service, `block` otherwise. |
 | `${.ip_policy.matching_rule}` | The rule that triggered an IP Policy match on the endpoint.                                   |
 
-### ngrok Template Variables {#ngrok-variables}
+#### ngrok Variables {#ngrok-variables}
```

Comment:
It seems like these variable sections are in alphabetical order except the ngrok variables have jumped ahead of Mutual TLS.

Reply:
fixed, ty.

@russorat unrelated low-priority item: it feels like these `.ngrok` scoped variables should be deprecated in favor of variables that already exist in other namespaces

`.ngrok.client_ip` -> `.conn.client_ip`
`.ngrok.request_id` -> `.http.request_id`

`.http.request_id` i think should become `.http.request.ngrok_id` as well to match the scoping of all of the other http request variables which are all nested under `.http.request`. right now `.request_id` is an outlier.

doing this migration should be easy. we should just be able to add these new paths and keep the old ones around but remove their public documentation